os.txt

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE `os` (
+  `aparelho` varchar(100) DEFAULT NULL,
+  `marca` varchar(100) DEFAULT NULL,
+  `modelo` varchar(100) DEFAULT NULL,
+  `serie` varchar(100) DEFAULT NULL,
+  `cor` varchar(100) DEFAULT NULL,
+  `acessorio` varchar(100) DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```

Comment:
Se for alterar alguma tabela, usar migrations ou criar aquivo sql na pasta updates.